src/main/java/juke/input/InputHandler.java

```diff
@@ -0,0 +1,54 @@
+package juke.input;
+
+import juke.task.TaskManager;
+import juke.main.Juke;
+import juke.main.Constants;
+
+public class InputHandler {
+    private static String textInput;
+
+    private InputHandler() {
+    }
+
+    public static void readText(TaskManager taskManager, String textInput) {
+        Parser.cleanText(textInput);
+        respondToText(taskManager, Parser.getInputCommand(), Parser.getTaskName(), Parser.getTaskIndex(),
+                Parser.getDeadline(), Parser.getFrom(), Parser.getTo());
+    }
+
+    //uses a switch to execute appropriate methods for the input
```

Comment:
Implementation details like this can be left out of the comments. This may not be providing any useful input to the reader.